docker-compose.migrate.yaml

```diff
@@ -0,0 +1,50 @@
+version: '3.8'
```

Comment:
This file orchestrates running of migrations locally and in the pipeline.  Basically you can copy and paste this from any repo and change the db and container names.  It works in combination with the scripts added to the `scripts` folder.